### DISCUSSION TITLE
Naming scheme for Encode to a Single File

### DISCUSSION THREAD

**Discussion by andybloke** [enzo1982/freac#428] [Help & Support] [answered]
Hi there!
My Output Filename pattern is set to album - albumartist\ disc - title - artist
However, when I encode to a single file, no folders are created and the output file is always albumartist - album and I can't seem to get it to change. Is this possible?

**Comment by enzo1982** [comment, marked as answer]
The default file name for single file encodes is not configurable, but is fixed as `<albumartist> - <album>`. The regular filename pattern is not used for single file encodes, because it usually contains elements that change with each track, like the `<title>` part in your example.

However, the latest [continuous build](https://github.com/enzo1982/freac/releases/continuous) adds a new tool called the _Output Splitting Tool_ that can be used to achieve this.

After loading your files into the joblist, select _Encode->Output splitting tool..._ or click the new toolbar button left to the _Start encoding_ button.

In the following dialog, either select _Split by duration_ and set a very long _Duration per part_, so you get only one output file. Or alternatively, select _Split upon metadata change_ and set the comparison pattern so that you get only one output file.

In both cases, you can set the desired filename pattern at the bottom of the dialog.

You can also use this in the _Split upon metadata change_ mode to convert multiple albums/discs to a single file per album/disc in one step.
